apps/web/pages/api/user/avatar.ts

```diff
@@ -29,7 +29,7 @@ async function getIdentityData(req: NextApiRequest) {
     };
   }
   if (teamname) {
-    const team = await prisma.team.findUnique({
+    const team = await prisma.team.findFirst({
       where: { slug: teamname },
       select: { logo: true },
```

Comment:
Same as above - we should probably tie this i with orgId

Reply:
Making this API subdomain-aware, so asking `acme.cal.com/api/user/avatar?teamname=engineering` will return the avatar for a team name engineering in acme org.